mipagina.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hola mundo</title>
```

Comment:
Agegar signo de admiración
```suggestion
    <title>Hola mundo!</title>
```